.github/workflows/release-brew.yml

```diff
@@ -31,27 +31,24 @@ jobs:
         node --version
         java -version
     - name: Check
-      run: dafny /compile:0 a.dfy
+      run: dafny verify a.dfy
     - name: Check bad
-      run: dafny /compile:0 b.dfy || echo EXPECTED FAILURE
+      run: dafny verify b.dfy || echo EXPECTED FAILURE
     ## Check that a simple program compiles and runs on each supported platform
     - name: Check C# compile
       run: |
-        dafny /compileVerbose:0 /compile:3 /compileTarget:cs /spillTargetCode:3 a.dfy
+        dafny run --target:cs a.dfy | tail -n +3 | diff expect.txt -
     - name: Check Go compile
       run: |
-        dafny /compile:3 /spillTargetCode:3 /compileTarget:go a.dfy
-        env GO111MODULE=auto GOPATH=$PWD/a-go go run a-go/src/a.go > actual.txt
-        diff expect.txt actual.txt
+        dafny run --target:go a.dfy | tail -n +3 | diff expect.txt -
```

Comment:
I'm not sure that these changes are desirable: do we have any other test that checks manual compilation? (I mean spill + compile, not `run`).

Reply:
Fair comment --- I reverted these changes. Whether to change these to new CLI and change to 'dafny run' is an independent question that can be decided separately.